terraform/cloudwatch.tf.json.template.py

```diff
@@ -15,6 +20,44 @@
 )
 
 
+def lambda_alarm_thresholds():
```

Comment:
- type hints would be helpful
- I assume there is a hidden uniqueness constraint, consider making it explicit by returning a dict
- look for a better place for this method or ask for suggestions in PL if you can't find a good one